src/common/POLFields/FieldsLocation/FieldsLocation.js

```diff
@@ -42,14 +51,15 @@ const FieldsLocation = ({ locations, disabled, isDisabledToChangePaymentInfo, wi
       renderField={(field) => (
         <Row>
           <Col xs={6}>
-            <FieldSelect
-              dataOptions={locations}
-              fullWidth
-              label={<FormattedMessage id="ui-orders.location.nameCode" />}
+            <FieldLocation
+              isDisabled={disabled}
+              labelId="ui-orders.location.nameCode"
+              locationsForDict={locations}
               name={`${field}.locationId`}
+              onChange={({ id }) => onChangeLocation(`${field}.locationId`, id)}
```

Comment:
what's about the comment? https://github.com/folio-org/ui-orders/pull/884#discussion_r437030678

Reply:
fixed